rosbag2_transport/test/rosbag2_transport/test_record_all.cpp

```diff
@@ -136,20 +139,71 @@ TEST_F(RecordIntegrationTestFixture, published_messages_from_multiple_services_a
   EXPECT_EQ(recorded_messages.size(), expected_messages);
 }
 
-TEST_F(RecordIntegrationTestFixture, published_messages_from_topic_and_service_are_recorded)
+TEST_F(RecordIntegrationTestFixture, published_messages_from_multiple_actions_are_recorded)
```

Comment:
maybe we can have another test case for cancel event? as far as i see, cancel event is not tested anywhere?

Reply:
Yes. I will add a test to check cancel event to be recorded.